<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovedFilmsWindow</class>
 <widget class="QDialog" name="MovedFilmsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moved films</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/window/moved-films</normaloff>:/window/moved-films</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
   <item>
    <widget class="QGroupBox" name="gbSeachMovedFilmsIn">
     <property name="title">
      <string>Search moved films in</string>
     </property>
     <layout class="QHBoxLayout" name="layoutDirectory">
      <item>
       <widget class="QLineEdit" name="eDirectory">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSelectDirectory">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cSearchInSubdirs">
     <property name="text">
      <string>Search in subdirectories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutScan" stretch="1,2,1">
     <item>
      <spacer name="spacerScanLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bScan">
       <property name="text">
        <string>&amp;Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerScanRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FoundedListWidget" name="gbFounded" native="true"/>
   </item>
   <item>
    <widget class="Line" name="lineMain0">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFooter" stretch="0,0,0,0,0,1,0,0,0">
     <item>
      <widget class="QLabel" name="labelTotalFounded">
       <property name="text">
        <string>Total founded:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTotalFounded">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineFooter0">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSelected">
       <property name="text">
        <string>Selected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSelected">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bMove">
       <property name="text">
        <string>&amp;Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoundedListWidget</class>
   <extends>QWidget</extends>
   <header>scanner/foundedlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>MovedFilmsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
